mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1355,7 +1359,7 @@ def predict_with_latency(X_copy):
             else:
                 self.y_probs = None
 
-        output_column_name = self.evaluator_config.get(_Y_PREDICTED_OUTPUT_COLUMN_NAME, "output")
+        output_column_name = self.predictions if self.predictions is not None else "output"
```

Comment:
I think lets throw an error instead if the prediction column is not specified for multi output. 
Defaulting to output is confusing to the user in the error message as to where this output column came from?

We can push the check of the multi-output into `_extract_output_and_other_columns` function.

Reply:
Updated